From f07b6ac6ced9e6a7dfb58630a563209f0dfc9c96 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Mon, 3 Nov 2014 16:07:28 +0100
Subject: [LIBREPORT PATCH] rhtsupport: be able to turn uReport off from GUI

Expose SubmitUReport configuration option in the GUI configuration.
Users should be able to configure entire reporting process from GUI.

Related to rhbz#1139987

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/report_RHTSupport.xml.in | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/src/plugins/report_RHTSupport.xml.in b/src/plugins/report_RHTSupport.xml.in
index db01d14..9c0755a 100644
--- a/src/plugins/report_RHTSupport.xml.in
+++ b/src/plugins/report_RHTSupport.xml.in
@@ -32,5 +32,22 @@
             <_description>Check SSL key validity</_description>
             <default-value>yes</default-value>
         </option>
+        <advanced-options>
+            <option type="bool" name="RHTSupport_SubmitUReport">
+                <_label>Submit uReport</_label>
+                <_note-html>Submit &lt;a href="https://access.redhat.com/articles/642323"&gt;micro-report&lt;/a&gt; when creating a new case. </_note-html>
+                <default-value>yes</default-value>
+            </option>
+            <option type="text" name="http_proxy">
+                <_label>HTTP Proxy</_label>
+                <allow-empty>yes</allow-empty>
+                <_note-html>Sets the proxy server to use for HTTP</_note-html>
+            </option>
+            <option type="text" name="HTTPS_PROXY">
+                <_label>HTTPS Proxy</_label>
+                <allow-empty>yes</allow-empty>
+                <_note-html>Sets the proxy server to use for HTTPS</_note-html>
+            </option>
+        </advanced-options>
     </options>
 </event>
-- 
1.8.3.1

